### ISSUE TITLE
Update model metadata README

### ISSUE THREAD

**Issue by rborchering** [cdcepi/FluSight-forecast-hub#2]
- Add schema_version if this should be included in each model metadata file. 
- Check update to model_abbr. Does team_abbr and model_abbr automatically get concatenated or should the full name be specified in the metadata?
-  Check syntax of model contributors field and add formatting for including orcid id
- Update accepted licenses link to somewhere in this repo? Or to the list provided in the model metadata schema?

**Comment by elray1** [comment]
Here are some comments on this:

- schema_version will not be included. We took it out on the fork's version of the metadata schema.
- The description of team_abbr and model_abbr looks ok to me. They will be tracked as separate fields in the model metadata file that are then concatenated.
- I submitted a pull request [here](https://github.com/cdcepi/FluSight-forecast-hub/pull/3) to update text related to the model contributors field formatting
- How about we link to the listin the model metatdata schema?
